src/MagnumPlugins/AssimpImporter/Test/Test.cpp

```diff
@@ -0,0 +1,257 @@
+                Assimp::Logger::Info|Assimp::Logger::Err|Assimp::Logger::Warn|Assimp::Logger::Debugging);
+    }
+
+    addTests({&AssimpImporterTest::open,
+
+              &AssimpImporterTest::camera,
+              &AssimpImporterTest::light,
+              &AssimpImporterTest::material,
+              &AssimpImporterTest::mesh,
+              &AssimpImporterTest::scene,
+              &AssimpImporterTest::texture});
+}
+
+void AssimpImporterTest::open() {
+    AssimpImporter importer;
+
+    auto data = Utility::Directory::read(Utility::Directory::join(ASSIMPIMPORTER_TEST_DIR, "scene.ogex"));
+    CORRADE_VERIFY(importer.openData(data));
+
+    importer.close();
+    CORRADE_VERIFY(!importer.isOpened());
+}
+
+void AssimpImporterTest::camera() {
+    AssimpImporter importer;
+    CORRADE_VERIFY(importer.openFile(Utility::Directory::join(ASSIMPIMPORTER_TEST_DIR, "camera.dae")));
+
+    CORRADE_COMPARE(importer.cameraCount(), 1);
+    std::optional<Trade::CameraData> camera = importer.camera(0);
+    CORRADE_VERIFY(camera);
+    CORRADE_COMPARE(camera->fov(), 49.13434_degf);
+    CORRADE_COMPARE(camera->near(), 0.123f);
+    CORRADE_COMPARE(camera->far(), 123.0f);
+
+    CORRADE_COMPARE(importer.object3DCount(), 2);
+
+    std::unique_ptr<Trade::ObjectData3D> cameraObject = importer.object3D(1);
+    CORRADE_COMPARE(cameraObject->instanceType(), ObjectInstanceType3D::Camera);
+    CORRADE_COMPARE(cameraObject->instance(), 0);
+}
+
+void AssimpImporterTest::light() {
+    AssimpImporter importer;
+    CORRADE_VERIFY(importer.openFile(Utility::Directory::join(ASSIMPIMPORTER_TEST_DIR, "light.dae")));
+
+    if(aiGetVersionMajor() <= 3 && aiGetVersionMinor() < 1) {
+        /* Cannot load lights with undefined light type yet */
+        CORRADE_COMPARE(importer.lightCount(), 3); /* 3 valid */
+    } else {
+        CORRADE_COMPARE(importer.lightCount(), 4); /* 3 valid + 1 invalid */
+    }
+    CORRADE_COMPARE(importer.object3DCount(), 5); /* root + 4 light objects */
+
+    constexpr Trade::LightData::Type types[3]{
+        Trade::LightData::Type::Spot,
+        Trade::LightData::Type::Point,
+        Trade::LightData::Type::Infinite};
+    constexpr Color3 colors[3]{
+        {0.12f, 0.24f, 0.36f},
+        {0.5f, 0.25f, 0.05f},
+        {1.0f, 0.15f, 0.45f}};
+
+    for (int i : {0, 1, 2}) {
+        std::optional<Trade::LightData> light = importer.light(i);
+        CORRADE_VERIFY(light);
+        CORRADE_COMPARE(light->type(), types[i]);
+        CORRADE_COMPARE(light->color(), colors[i]);
+        CORRADE_COMPARE(light->intensity(), 1.0f);
+
+
+        std::unique_ptr<Trade::ObjectData3D> lightObject = importer.object3D(i + 1);
+        CORRADE_COMPARE(lightObject->instanceType(), ObjectInstanceType3D::Light);
+        CORRADE_COMPARE(lightObject->instance(), i);
+    }
+
+    if(aiGetVersionMajor() <= 3 && aiGetVersionMinor() < 1) {
+        /* Cannot load lights with undefined light type yet */
+        return;
```

Comment:
I would use `CORRADE_SKIP()` here so it's visible in the output. Also, like I mentioned on Gitter, use some combined version integer for comparison (this would fail for theoretical version 2.9 for example).